src/app/branches/page.tsx

```diff
@@ -1,16 +1,13 @@
 import Navbar from "@/components/shared/Navbar/Navbar";
 import Footer from "@/components/shared/Footer/Footer";
+import BranchPage from "@/components/landing/body/BranchSection/BranchPage";
 
 export default function Home() {
   return (
     <main>
       <div className="flex flex-col align-center">
         <Navbar />
-
-        {/* STUB PAGE */}
-        <h1 className="justify-self-center">STUB PAGE</h1>
-        {/* replace this ^ with an actual page */}
-
+        <BranchPage />
```

Comment:
No need to create a separate BranchPage component — since it'll get used in one place, might as well keep it within the page.tsx file